test/oli_web/live/sections/enrollments_view_test.exs

```diff
@@ -74,6 +75,27 @@ defmodule OliWeb.Sections.EnrollmentsViewTest do
       {:ok, _view, html} =
         live(conn, Routes.live_path(@endpoint, OliWeb.Sections.EnrollmentsView, section.slug))
 
+      assert html =~ "Admin"
+
```

Comment:
General comment throughout these files / changes - what's your thoughts about removing blank lines between assertions?

Reply:
It's the auto format on save that leaves it like that. Maybe I can see if is a specific rule I can change